modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro

```diff
@@ -47,9 +47,16 @@ definition {
 	}
 
 	macro assertDeleteEntryButtonNotPresent {
-		AssertElementNotPresent(
-			key_entry = "${entry}",
-			locator1 = "ObjectPortlet#ENTRY_KEBAB_MENU");
+		if (IsElementPresent(key_entry = "", locator1 = "ObjectPortlet#ENTRY_KEBAB_MENU")) {
+			Click(
+				key_entry = "${entry}",
+				locator1 = "ObjectPortlet#ENTRY_KEBAB_MENU");
+		}
+		else {
+			AssertElementNotPresent(
+				key_entry = "${entry}",
+				locator1 = "ObjectPortlet#ENTRY_KEBAB_MENU");
+		}
```

Comment:
Hi @thaiscabrals Your vision for covering different behaviors is great and your changes leave the macro with more coverage, that's good, but we don't use assertions in macros. All assertions must be on the test level.

Please consult our [Test Review Guidelines](https://liferay.atlassian.net/wiki/spaces/DXPL/pages/2114356408/Test+Review+Guidelines#:~:text=Verifying%20assertions%20are,the%20test%20level.) to more information

I have two points where you could work:
1 - When making a change in a macro in the fix of a test, you can add more clear commit messages. You can see an example [here](https://liferay.atlassian.net/wiki/spaces/DXPL/pages/2114356408/Test+Review+Guidelines#:~:text=Adding%20unclear%20commit%20messages.)

2 - If you make any changes to the macro you'll need to see if your changes affect another test

Reply:
Hi @evanilson-santana I did not add Assertions to the macro, it already existed. I added a click so that when there is a kebab, it will be clicked.
And since the macro was modified to cover two different behaviors, od two behaviors were tested. Next time I will specify this better.
Thank you for your review.